dataset/dictionary/characters.json5

```diff
@@ -776,7 +807,8 @@
     ja: "天権",
     zhCN: "天权",
     pronunciationJa: "テンチュエン",
-    notes: "凝光の称号",
+    notes: "凝光の称号、北斗七星の天権星に由来します、文曲星とも呼ばれる。 [Wikipedia: おおぐま座デルタ星](https://ja.wikipedia.org/wiki/おおぐま座デルタ星)",
```

Comment:
> I think [北斗七星 on Wikipedia](https://ja.wikipedia.org/wiki/%E5%8C%97%E6%96%97%E4%B8%83%E6%98%9F#%E5%8C%97%E6%96%97%E4%B8%83%E6%98%9F%E3%82%92%E6%A7%8B%E6%88%90%E3%81%99%E3%82%8B%E6%98%9F) is better as Japanese reference URL because there is no explanation for 天権星 in [the article of おおぐま座デルタ星](https://ja.wikipedia.org/wiki/%E3%81%8A%E3%81%8A%E3%81%90%E3%81%BE%E5%BA%A7%E3%83%87%E3%83%AB%E3%82%BF%E6%98%9F). In the [article of 北斗七星], there is a list of all the Chinese names of each star in the Big Dipper.

Thanks, I think this can be better.

> Regarding 文曲星, I think we don't have to mention it here because it is not related to Ningguang or other Genshin Impact terms.

Regarding to it, 文曲星 is a highly common known word in Chinese, and people may relate the meaning of 文曲星 to the characteristic of Ningguang herself. If you think this word is not common in Japanese and seldom people know it, let's just delete it.

Reply:
> Regarding to it, 文曲星 is a highly common known word in Chinese

OK, then let's keep the topic about 文曲星 only in the Chinese note and delete it from the Japanese note.